traffic_ops/app/templates/not_found.html.ep

```diff
@@ -27,7 +27,7 @@
 		  </div>
 
 			<div style="margin-top:20px;clear:both;">
-					If you are looking for a web page, the old Traffic Ops user interface was removed in Traffic Ops 4.0. Please use the <a href="https://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_portal/usingtrafficportal.html#usingtrafficportal">Traffic Portal</a> instead.
+					If you are looking for a web page, the old Traffic Ops user interface was removed in Traffic Ops 4.0. Please use the <a href="<%= $config->{'portal'}{'docs_url'} %>"> Traffic Portal</a> instead.
```

Comment:
It seems like every CDN operator is going to immediately set this to the link to their Traffic Portal, I can't imagine why anyone _wouldn't_ link there, and thus have `"docs_url" : "https://trafficops.example.net"` in their config with this name.

Would you object to changing the config from `docs_url` to something like `traffic_portal_deprecation_link_uri`? That way it makes sense as the docs which it defaults to in the repo, but also isn't confusing to operators when it's set to the instance URI?